packages/meteor-accounts-saml/saml_server.js

```diff
@@ -5,7 +5,7 @@ import _ from 'underscore';
 if (!Accounts.saml) {
 	Accounts.saml = {
 		settings: {
-			debug: true,
+			debug: false,
```

Comment:
Can you transform this in a setting? That way it's possible to enable it in production env

Reply:
Done